src/app/tests/TestAttributeValueEncoder.cpp

```diff
@@ -238,6 +239,58 @@ void TestEncodeEmptyList(nlTestSuite * aSuite, void * aContext)
     VERIFY_BUFFER_STATE(aSuite, test, expected);
 }
 
+void TestEncodeFabricScoped(nlTestSuite * aSuite, void * aContext)
+{
+    TestSetup test(aSuite, 1);
```

Comment:
add a comment on what `1` means in the above (like ` /* fabricIndex = */ 1` or `1 /* fabricIndex */` ... assuming 1 is fabricindex). It was not clear to  me how fabric index filgering and test encoder interact with each other.

Reply:
Replaced by a const expr.